ml/ml.cc

```diff
@@ -138,6 +138,50 @@ char *ml_get_anomaly_event_info(RRDHOST *RH, const char *AnomalyDetectorName,
     return strdup(Json.dump(4, '\t').c_str());
 }
 
+char *ml_get_anomaly_rate_info(RRDHOST *RH, time_t After, time_t Before) {
```

Comment:
A dummy definition that ignores its arguments, is needed in `ml/ml-dummy.c`

Reply:
Defined.